### ISSUE TITLE
Cannot import in ember CLI

### ISSUE THREAD

**Issue by mihai-scurtu** [Blooie/ember-radio-buttons#7]
When trying to export the component using

```
import EmberRadioButton from 'ember-radio-buttons';

export default EmberRadioButton;
```

I get the error

```
Uncaught Error: Could not find module ember-radio-buttons
```

I will manually add the source code in for now.

**Comment by mihai-scurtu** [comment]
Nevermind, restarting the server fixed it.

**Comment by mansona** [comment]
:+1:

**Comment by dmuneras** [comment]
Hi @mansona  and @mihai-scurtu 

I was using an older version of the addon, I updated to 3.0.1 to be able to set the disabled property. But I am having the same problem. I created a component using 'ember generate component radio-button' and add the code you put in the README in the component, but I am getting the same error.

I restart the server... reinstall npm modules....

Thanks in advance,
Daniel.
